packages/assistant-stream/src/core/utils/stream/AssistantTransformStream.ts

```
@@ -0,0 +1,79 @@
+  controller: AssistantTransformStreamController,
+) => void | PromiseLike<void>;
+
+type AssistantTransformerStartCallback = (
+  controller: AssistantTransformStreamController,
+) => void | PromiseLike<void>;
+
+interface AssistantTransformerTransformCallback<I> {
+  (
+    chunk: I,
+    controller: AssistantTransformStreamController,
+  ): void | PromiseLike<void>;
```

Biome (1.9.4)

[error] Use a function type instead of a call signature.

Types containing only a call signature can be shortened to a function type.
Safe fix: Alias a function type instead of using an interface with a call signature.


(lint/style/useShorthandFunctionType)